hw02-collections/src/main/java/ru/otus/CustomerService.java

```diff
@@ -0,0 +1,39 @@
+package ru.otus;
+
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class CustomerService {
+
+    TreeMap<Customer, String> treeMap =
+            new TreeMap<>(Comparator.comparing(Customer::getScores));
```

Comment:
```suggestion
    private Map<Customer, String> customerScores  =
            new TreeMap<>(Comparator.comparing(Customer::getScores));
```

Поля всегда делайте приватными и лучше объявлять как интерфейсы.